launchlib/const.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2016 Palantir Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package launchlib
+
+import (
+	"fmt"
+)
+
+const (
+	LauncherStaticFile = "service/bin/launcher-static.yml"
+	LauncherCustomFile = "var/conf/launcher-custom.yml"
+	OutputFileFormat   = "var/log/%sstartup.log"
```

Comment:
If we include the dash, then the main process's log file gets named `var/log/-startup.log`. Note how the subprocess log files get named `fmt.Sprintf(OutputFileFormat, "name-")`. I can see how the dash in the `Sprintf` could be confusing, though, could make a small wrapper function for this?

Reply:
I think the format should specifically be for secondary processes and a separate variable should explicitly declare the main processes output file, since it's static.